src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -5957,6 +5957,30 @@ && isInternalCase(asylumCase)
         );
     }
 
+    @Bean
+    public PreSubmitCallbackHandler<AsylumCase> internalNonStandardDirectionAppellantLetterNotificationHandler(
+            @Qualifier("internalNonStandardDirectionAppellantLetterNotificationGenerator")
+            List<NotificationGenerator> notificationGenerators,
+            DirectionFinder directionFinder) {
+
+        return new NotificationHandler(
+                (callbackStage, callback) -> {
+
+                    AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+                    YesOrNo appellantHasFixedAddress = asylumCase.read(APPELLANT_HAS_FIXED_ADDRESS, YesOrNo.class).orElse(NO);
```

Comment:
This will only check if the appellantHasFixedAddres and the fixed addres is for inCountry appeals only,
You can use this one instead

`
hasAppellantAddressInCountryOrOutOfCountry(asylumCase)
`
Also, please test this notification for OOC appeals